test/salt/pillar/exim.sls

```diff
@@ -13,3 +13,9 @@ exim:
   mailname: foo.bar.baz
   aliases:
     root: root@bar.baz
+  files:
+    base:
+      'passwd.client': |
+        # <server>:<username>:<user password>
+        *:mymail@example.com:abdc1243
+
```

Comment:
```suggestion
```

Tripping the `yamllint`.

* https://travis-ci.com/github/saltstack-formulas/exim-formula/jobs/329640541#L292-L296